web-ui/src/main/resources/catalog/components/viewer/wfsfilter/WfsFilterDirective.js

```diff
@@ -110,24 +110,29 @@
     'wfsFilterService',
     '$q',
     '$rootScope',
+    '$translate',
     'gnIndexRequestManager',
     'gnIndexService',
     'gnGlobalSettings',
     'gnSearchSettings',
     'gnFeaturesTableManager',
-    function($http, wfsFilterService, $q, $rootScope,
+    'gnHttp',
+    'gnAlertService',
+    'gnFacetTree',
+    function($http, wfsFilterService, $q, $rootScope, $translate,
              gnIndexRequestManager, gnIndexService, gnGlobalSettings,
-             gnSearchSettings, gnFeaturesTableManager) {
+             gnSearchSettings, gnFeaturesTableManager, gnHttp,
+             gnAlertService, gnFacetTree) {
       return {
         restrict: 'A',
         replace: true,
         templateUrl: '../../catalog/components/viewer/wfsfilter/' +
-            'partials/wfsfilterfacet.html',
+          'partials/wfsfilterfacet.html',
         scope: {
           featureTypeName: '@',
           wfsUrl: '@',
           displayCount: '@',
-          layer: '='
+          baseLayer: '=layer'
```

Comment:
Same here, looks to be used in Sextant for something else

```

          // SEXTANT SPECIFIC
          // use nested wms if we're in a group
          if (scope.baseLayer.get('originalWms')) {
            scope.layer = scope.baseLayer.get('originalWms');
          } else {
            scope.layer = scope.baseLayer
          }
          // END SEXTANT SPECIFIC
```

Currently layer is never set ... maybe we should add some config to handle both strategies.

Reply:
Added both. Sextant can set baseLayer if needed which takes priority.